### DISCUSSION TITLE
Brandon Williams & Stephen Celis OnLine 1 day

### DISCUSSION THREAD

**Discussion by formacion-tyris** [pointfreeco/swift-composable-architecture#1961] [Ideas]
Is possible that Brandon Williams & Stephen Celis will make a on line discussion?

**Comment by oliverfoggin** [comment]
I'm not sure exactly what you mean by an "online discussion"? Is that not what the discussions are for on here?

But also, if you don't know about it, there is a new PointFree Slack workspace that you can join to chat with the community also.

You can read about it here... https://www.pointfree.co/blog/posts/100-point-free-turns-5
